README.md

```diff
@@ -386,6 +386,40 @@ Note that dotdrop uses different delimiters than
 * comment start = `{#@@`
 * comment end = `@@#}`
 
+## Available variables
+### Profile
+
+`{{@@ profile @@}}` contains the profile being used. Below in the example you can see how this is useful.
+
+### Environment variables
+
+It's possible to access environment variables inside the templates. You can utilize this functionality like this:
```

Comment:
~~You can utilize this functionality like this~~ -> *This feature can be used like this*